test/snippet/range/view/single_pass_input.cpp

```diff
@@ -1,18 +1,16 @@
-#include <iostream>
+#include <string>
 
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/range/view/single_pass_input.hpp>
 
-using namespace seqan3;
-
 int main()
 {
-//! [usage]
-std::string str{"hello"};
-auto v = str | view::single_pass_input;
-debug_stream << *++v.begin() << std::endl;  // prints 'e'
-debug_stream << *++v.begin() << std::endl;  // prints 'l'
-debug_stream << *++v.begin() << std::endl;  // prints 'l'
-debug_stream << *++v.begin() << std::endl;  // prints 'o'
-//! [usage]
+    std::string str{"hello"};
+    auto v = str | seqan3::view::single_pass_input;
+
+    seqan3::debug_stream << *v.begin() << std::endl;    // prints 'h'
+    seqan3::debug_stream << *++v.begin() << std::endl;  // prints 'e'
+    seqan3::debug_stream << *++v.begin() << std::endl;  // prints 'l'
+    seqan3::debug_stream << *++v.begin() << std::endl;  // prints 'l'
+    seqan3::debug_stream << *++v.begin() << std::endl;  // prints 'o'
```

Comment:
replace endl with `\n`

Reply:
wouldn't this be a littler more readable:
```cpp
    std::string str{"hello"};
    auto v = str | seqan3::view::single_pass_input;
    auto b = v.begin();

    seqan3::debug_stream << *b << std::endl;    // prints 'h'
    seqan3::debug_stream << *(++b) << std::endl;  // prints 'e'
    seqan3::debug_stream << *(++b) << std::endl;  // prints 'l'
    seqan3::debug_stream << *(++b) << std::endl;  // prints 'l'
    seqan3::debug_stream << *(++b) << std::endl;  // prints 'o'
```

At least put () around the interior expression